src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  padding: 10px 0;
+  display: block;
+  height: 100%;
+  width: 100%;
+}
+
+.navbar {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__item a {
+  padding: 23px 0;
```

Comment:
no need to align vertically , you have achieved that in line 38 with `align-items: center;`